components/core/core.module.ts

```diff
@@ -1,3 +1,4 @@
+import * as merge from 'deepmerge';
```

Comment:
We have [structuredClone()](https://github.com/hslayers/hslayers-ng/blob/32cd15abeb92ee121c831c24e192c6b31d1570b2/components/utils/utils.service.ts#L329) in our UtilsService, in case it can be helpful

Reply:
Thanks, I will try. Would be good to leave out a dependency

Reply:
On the second hand, it's just 700bytes compressed and merge.all does more than our utils function and more elegantly. I think I'm gonna keep it after all :)

Reply:
@raitisbe If it is more powerful and we will included it in out dependencies, then we can probably remove our custom function and replace its occurances with the deepmerge version - for consistency and easier debugging. What do you think?

Reply:
@jmacura Was afraid to break something and was in a rush, but I agree.